app/services/notification_service.py

```
@@ -235,15 +245,63 @@ async def send_danger_notification(self, request: DangerNotificationRequest) ->
+            if message:
+                body += f" 메시지: {message}"
+
+            success = await self.fcm_pusher.send_notification(
+                device_token=device_token,
+                body=body,
+                from_user_id=from_user_id,
+                level=level,
+            )
+            print(f"[FCM] Notification sent to {device_token}: {success}")
+            return success
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)